### PR TITLE
Fix navigation 

### PR DESCRIPTION
**Pull request by neelxie**
# Description
It fixes issues of routing using our sidebar especially traversing app-related metrics, also integrates css modules for the component.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Trello Ticket ID
https://trello.com/c/0W7uTm37

## How Can This Been Tested?

Run the branch and test out the sidebar for all resources.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

# Screenshots
**Before**
Screenshot from 2021-07-14 22-22-27

**After**
Screenshot from 2021-07-14 22-00-28

### CODE REVIEW

**Comment by fel-w** [comment]
How is the idea of maintaining `Apps` as a `SideBar_SubBarListItem` under `DASHBOARD` as the `SideBar_ListItem`? It makes it more consistent since `Dashboard` opens `Apps` yet other links like `Databases` open `Databases`.

image


image


image

**Comment by neelxie** [comment]
> How is the idea of maintaining `Apps` as a `SideBar_SubBarListItem` under `DASHBOARD` as the `SideBar_ListItem`? It makes it more consistent since `Dashboard` opens `Apps` yet other links like `Databases` open `Databases`.
> 
> **Consistent**
> image
> 
> **Inconsistent**
> image
> 
> **Proposed Change**
> image

@fel-w thank you for this, @Mubangizi what's your take on this?

**Comment by neelxie** [comment]
Add fix for project name in sidebar for databases.
Screenshot from 2021-07-21 18-53-02

File: src/components/SideBar/SideBar.module.css [hunk 1]
```diff
@@ -84,3 +84,17 @@
...
+  font-size: 12px;
+  line-height: 14px;
+  bottom: 0;
+  color: #008ac1;
+  padding: 18px 0 20px 18px;
+  cursor: pointer;
+  background-color: #06222f;
+}
+.SideFooter:hover {
+  color: rgb(131, 182, 202);
```
Comment (kawerewagaba) [inline src/components/SideBar/SideBar.module.css:99]: still have issues with `rgb` if we don't need an alpha channel (transparency). let's stick with `hex`

**Review by kawerewagaba — APPROVED** [review]

**Pull request merged**